test/dummy/db/migrate/20130125101430_create_users.rb

```diff
@@ -1,4 +1,4 @@
-class CreateUsers < ActiveRecord::Migration
+class CreateUsers < ActiveRecord::Migration[5.0]
```

Comment:
What happens to a Rails 4.2 app if you do this? Will it break?

Reply:
The changes in the library should not affect Rails 4. This change would imho break a 4.x app however here it's only a part of tests.

Reply:
Ahh, good point, thanks!